rest/mgmt_storage/2019-06-01/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
My initial reaction is that we should _not_ have this generated code in different crates but as part of the crate where they will be used.